src/themes/clean/templates/elements/journal/issue_sidebar.html

```diff
@@ -20,7 +20,7 @@ <h2>
                 {% for issue in issues %}
                 {% if issue.issue_type.code == 'issue' %}
                     <li><a href="{% url 'journal_issue' issue.id %}" aria-label='{{ issue.display_title_a11y }}'>
-                        {{ issue.display_title }} 
+                        {{ issue.display_title }}
```

Comment:
Try to avoid formatting changes that are not related to the issues we are addressing